### ISSUE TITLE
Latest update prevents me from opening any project

### ISSUE THREAD

**Issue by rrajkomar** [zielu/GitToolBox#36]
When starting PHPStorm with the lastest plugin version, my currently opened project gets automatically closed and I'm sent  to the home screen. From there, I can't open any project unless I disable the plugin and restart.

**Comment by JosefKaser** [comment]
Same for me with PyCharm 2016.3.2, plugin version is 163.3. Disabling the plugin lets me open the project. But I need the plugin features for my daily work.

**Comment by JosefKaser** [comment]
Switching back to plugin version 163.2.1 by downloading it from [https://plugins.jetbrains.com/plugin/7499-gittoolbox](https://plugins.jetbrains.com/plugin/7499-gittoolbox) and installing the plugin from disk solved the problem for the moment.

**Comment by DrColossos** [comment]
had the same issue, disabling the plugin solved it for me. PHPStorm 2016.3.3

**Comment by sgr-amboss** [comment]
Here is part of my idea.log file that seems to contain the exception (I am on Ubuntu 16.04), if that is any help

```
2017-03-17 09:15:26,823 [  11980]   INFO - roject.impl.ProjectManagerImpl - com.intellij.diagnostic.PluginException: com/intellij/compiler/server/BuildManagerListener [Plugin: zielu.gittoolbox] 
com.intellij.ide.plugins.PluginManager$StartupAbortedException: com.intellij.diagnostic.PluginException: com/intellij/compiler/server/BuildManagerListener [Plugin: zielu.gittoolbox]
	at com.intellij.ide.plugins.PluginManager.handleComponentError(PluginManager.java:249)
	at com.intellij.openapi.components.impl.PlatformComponentManagerImpl.handleInitComponentError(PlatformComponentManagerImpl.java:43)
	at com.intellij.openapi.components.impl.ComponentManagerImpl$ComponentConfigComponentAdapter.getComponentInstance(ComponentManagerImpl.java:519)
	at com.intellij.openapi.components.impl.ComponentManagerImpl.createComponents(ComponentManagerImpl.java:125)
	at com.intellij.openapi.components.impl.ComponentManagerImpl.init(ComponentManagerImpl.java:109)
	at com.intellij.openapi.components.impl.ComponentManagerImpl.init(ComponentManagerImpl.java:96)
	at com.intellij.openapi.project.impl.ProjectImpl.init(ProjectImpl.java:287)
	at com.intellij.openapi.project.impl.ProjectManagerImpl.a(ProjectManagerImpl.java:222)
	at com.intellij.openapi.project.impl.ProjectManagerImpl.b(ProjectManagerImpl.java:463)
	at com.intellij.openapi.progress.impl.CoreProgressManager$5.run(CoreProgressManager.java:235)
	at com.intellij.openapi.progress.impl.CoreProgressManager$TaskRunnable.run(CoreProgressManager.java:710)
	at com.intellij.openapi.progress.impl.CoreProgressManager$11.run(CoreProgressManager.java:423)
	at com.intellij.openapi.progress.impl.CoreProgressManager$3.run(CoreProgressManager.java:179)
	at com.intellij.openapi.progress.impl.CoreProgressManager.a(CoreProgressManager.java:568)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:519)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:54)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:164)
	at com.intellij.openapi.application.impl.ApplicationImpl.a(ApplicationImpl.java:572)
	at com.intellij.openapi.application.impl.ApplicationImpl$2.run(ApplicationImpl.java:309)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: com.intellij.diagnostic.PluginException: com/intellij/compiler/server/BuildManagerListener [Plugin: zielu.gittoolbox]
	... 24 more
Caused by: java.lang.NoClassDefFoundError: com/intellij/compiler/server/BuildManagerListener
	at zielu.gittoolbox.fetch.AutoFetchState.initComponent(AutoFetchState.java:46)
	at com.intellij.openapi.components.impl.ComponentManagerImpl$ComponentConfigComponentAdapter.getComponentInstance(ComponentManagerImpl.java:501)
	... 21 more
Caused by: java.lang.ClassNotFoundException: com.intellij.compiler.server.BuildManagerListener PluginClassLoader[zielu.gittoolbox, 163.3]
	at com.intellij.ide.plugins.cl.PluginClassLoader.loadClass(PluginClassLoader.java:65)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	... 23 more
2017-03-17 09:15:26,826 [  11983]  ERROR - m.PlatformProjectOpenProcessor - com.intellij.ide.plugins.PluginManager$StartupAbortedException: com.intellij.diagnostic.PluginException: com/intellij/compiler/server/BuildManagerListener [Plugin: zielu.gittoolbox] 
java.io.IOException: com.intellij.ide.plugins.PluginManager$StartupAbortedException: com.intellij.diagnostic.PluginException: com/intellij/compiler/server/BuildManagerListener [Plugin: zielu.gittoolbox]
	at com.intellij.openapi.project.impl.ProjectManagerImpl.e(ProjectManagerImpl.java:473)
	at com.intellij.openapi.project.impl.ProjectManagerImpl.convertAndLoadProject(ProjectManagerImpl.java:449)
	at com.intellij.platform.PlatformProjectOpenProcessor.doOpenProject(PlatformProjectOpenProcessor.java:184)
	at com.intellij.ide.RecentDirectoryProjectsManager.doOpenProject(RecentDirectoryProjectsManager.java:55)
	at com.intellij.ide.ReopenProjectAction.actionPerformed(ReopenProjectAction.java:68)
	at com.intellij.openapi.wm.impl.welcomeScreen.RecentProjectPanel$2.onClick(RecentProjectPanel.java:126)
	at com.intellij.ui.ClickListener$1.mouseReleased(ClickListener.java:73)
	at java.awt.AWTEventMulticaster.mouseReleased(AWTEventMulticaster.java:290)
	at java.awt.AWTEventMulticaster.mouseReleased(AWTEventMulticaster.java:289)
	at java.awt.AWTEventMulticaster.mouseReleased(AWTEventMulticaster.java:289)
	at java.awt.Component.processMouseEvent(Component.java:6533)
	at javax.swing.JComponent.processMouseEvent(JComponent.java:3324)
	at com.intellij.openapi.wm.impl.welcomeScreen.RecentProjectPanel$MyList.processMouseEvent(RecentProjectPanel.java:347)
	at java.awt.Component.processEvent(Component.java:6298)
	at java.awt.Container.processEvent(Container.java:2236)
	at java.awt.Component.dispatchEventImpl(Component.java:4889)
	at java.awt.Container.dispatchEventImpl(Container.java:2294)
	at java.awt.Component.dispatchEvent(Component.java:4711)
	at java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4888)
	at java.awt.LightweightDispatcher.processMouseEvent(Container.java:4525)
	at java.awt.LightweightDispatcher.dispatchEvent(Container.java:4466)
	at java.awt.Container.dispatchEventImpl(Container.java:2280)
	at java.awt.Window.dispatchEventImpl(Window.java:2746)
	at java.awt.Component.dispatchEvent(Component.java:4711)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:758)
	at java.awt.EventQueue.access$500(EventQueue.java:97)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.awt.EventQueue$3.run(EventQueue.java:703)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:90)
	at java.awt.EventQueue$4.run(EventQueue.java:731)
	at java.awt.EventQueue$4.run(EventQueue.java:729)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:728)
	at com.intellij.ide.IdeEventQueue.g(IdeEventQueue.java:843)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:675)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:391)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
Caused by: com.intellij.ide.plugins.PluginManager$StartupAbortedException: com.intellij.diagnostic.PluginException: com/intellij/compiler/server/BuildManagerListener [Plugin: zielu.gittoolbox]
	at com.intellij.ide.plugins.PluginManager.handleComponentError(PluginManager.java:249)
	at com.intellij.openapi.components.impl.PlatformComponentManagerImpl.handleInitComponentError(PlatformComponentManagerImpl.java:43)
	at com.intellij.openapi.components.impl.ComponentManagerImpl$ComponentConfigComponentAdapter.getComponentInstance(ComponentManagerImpl.java:519)
	at com.intellij.openapi.components.impl.ComponentManagerImpl.createComponents(ComponentManagerImpl.java:125)
	at com.intellij.openapi.components.impl.ComponentManagerImpl.init(ComponentManagerImpl.java:109)
	at com.intellij.openapi.components.impl.ComponentManagerImpl.init(ComponentManagerImpl.java:96)
	at com.intellij.openapi.project.impl.ProjectImpl.init(ProjectImpl.java:287)
	at com.intellij.openapi.project.impl.ProjectManagerImpl.a(ProjectManagerImpl.java:222)
	at com.intellij.openapi.project.impl.ProjectManagerImpl.b(ProjectManagerImpl.java:463)
	at com.intellij.openapi.progress.impl.CoreProgressManager$5.run(CoreProgressManager.java:235)
	at com.intellij.openapi.progress.impl.CoreProgressManager$TaskRunnable.run(CoreProgressManager.java:710)
	at com.intellij.openapi.progress.impl.CoreProgressManager$11.run(CoreProgressManager.java:423)
	at com.intellij.openapi.progress.impl.CoreProgressManager$3.run(CoreProgressManager.java:179)
	at com.intellij.openapi.progress.impl.CoreProgressManager.a(CoreProgressManager.java:568)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:519)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:54)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:164)
	at com.intellij.openapi.application.impl.ApplicationImpl.a(ApplicationImpl.java:572)
	at com.intellij.openapi.application.impl.ApplicationImpl$2.run(ApplicationImpl.java:309)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: com.intellij.diagnostic.PluginException: com/intellij/compiler/server/BuildManagerListener [Plugin: zielu.gittoolbox]
	... 24 more
Caused by: java.lang.NoClassDefFoundError: com/intellij/compiler/server/BuildManagerListener
	at zielu.gittoolbox.fetch.AutoFetchState.initComponent(AutoFetchState.java:46)
	at com.intellij.openapi.components.impl.ComponentManagerImpl$ComponentConfigComponentAdapter.getComponentInstance(ComponentManagerImpl.java:501)
	... 21 more
Caused by: java.lang.ClassNotFoundException: com.intellij.compiler.server.BuildManagerListener PluginClassLoader[zielu.gittoolbox, 163.3]
	at com.intellij.ide.plugins.cl.PluginClassLoader.loadClass(PluginClassLoader.java:65)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	... 23 more
2017-03-17 09:15:26,828 [  11985]  ERROR - m.PlatformProjectOpenProcessor - PhpStorm 2016.3.3  Build #PS-163.13906.21 
2017-03-17 09:15:26,828 [  11985]  ERROR - m.PlatformProjectOpenProcessor - JDK: 1.8.0_112-release 
2017-03-17 09:15:26,828 [  11985]  ERROR - m.PlatformProjectOpenProcessor - VM: OpenJDK 64-Bit Server VM 
2017-03-17 09:15:26,828 [  11985]  ERROR - m.PlatformProjectOpenProcessor - Vendor: JetBrains s.r.o 
2017-03-17 09:15:26,828 [  11985]  ERROR - m.PlatformProjectOpenProcessor - OS: Linux 
2017-03-17 09:15:26,828 [  11985]  ERROR - m.PlatformProjectOpenProcessor - Last Action:  
2017-03-17 09:15:26,847 [  12004]   INFO - ellij.project.impl.ProjectImpl - 23 project components initialized in 15 ms 
2017-03-17 09:15:26,847 [  12004]   INFO - le.impl.ModuleManagerComponent - 0 module(s) loaded in 0 ms 
2017-03-17 09:15:35,268 [  20425]   INFO - org.jetbrains.io.BuiltInServer - web server stopped 
2017-03-17 09:15:35,269 [  20426]   INFO - Types.impl.FileTypeManagerImpl - FileTypeManager: 0 auto-detected files
Elapsed time on auto-detect: 0 ms 
2017-03-17 09:15:35,272 [  20429]   INFO - stubs.SerializationManagerImpl - START StubSerializationManager SHUTDOWN 
2017-03-17 09:15:35,273 [  20430]   INFO - stubs.SerializationManagerImpl - END StubSerializationManager SHUTDOWN 
2017-03-17 09:15:35,273 [  20430]   INFO - il.indexing.FileBasedIndexImpl - START INDEX SHUTDOWN 
2017-03-17 09:15:35,306 [  20463]   INFO - il.indexing.FileBasedIndexImpl - END INDEX SHUTDOWN 
2017-03-17 09:15:35,313 [  20470]   INFO - pl.local.NativeFileWatcherImpl - Watcher terminated with exit code 0 
2017-03-17 09:15:35,313 [  20470]   INFO - newvfs.persistent.PersistentFS - VFS dispose started 
2017-03-17 09:15:35,331 [  20488]   INFO - newvfs.persistent.PersistentFS - VFS dispose completed 
2017-03-17 09:15:35,334 [  20491]   INFO -        #com.intellij.idea.Main - ------------------------------------------------------ IDE SHUTDOWN 
```

**Comment by zielu** [comment]
Thanks for stacktrace, I'll fix this during weekend

**Comment by zielu** [comment]
Note to self: http://www.jetbrains.org/intellij/sdk/docs/basics/getting_started/plugin_compatibility.html

**Comment by arcticlinux** [comment]
+1

**Comment by kevincam3** [comment]
Same here. I had to disable the plugin so I could use PhpStom normally

**Comment by Topherross** [comment]
+1 MacBook Pro 2016 running OS Sierra. All JetBrains Editors are broken.

**Comment by st0rmchild** [comment]
Confirm working Version 163.3.1  in PyCharm Pro and Gogland EAP.
Thanks

**Comment by Topherross** [comment]
Newest version fixed the issue. Thank you for the quick fix.

**Comment by JosefKaser** [comment]
Works here with PyCharm Community 2016.3.2 and 2016.3.3. Thanks and thumbs up for the quick fix.

**Comment by zielu** [comment]
Fixed in 163.3.1 and 171.1.1
